aten/src/ATen/Context.cpp

```diff
@@ -282,9 +282,9 @@ at::BlasBackend Context::blasPreferredBackend() {
 #ifdef USE_ROCM
   if (blas_preferred_backend == at::BlasBackend::Cublaslt) {
     static const bool hipblaslt_unsupported = []() {
-      static const std::vector<std::string> archs = {"gfx90a", "gfx940", "gfx941", "gfx942"};
+      static const std::vector<std::string> archs = {"gfx90a", "gfx940", "gfx941", "gfx942", "gfx1200", "gfx1201"};
       for (auto index: c10::irange(getNumGPUs())) {
-        if (!detail::getCUDAHooks().isGPUArch(index, archs)) {
+        if (!detail::getCUDAHooks().isGPUArch(static_cast<c10::DeviceIndex>(index), archs)) {
```

Comment:
More for my own enlightenment, why did we need to add the static_cast?

Reply:
The linter complained about unsigned to signed cast here and the linter made this suggestion.  I blindly followed.